packages/test-commons/src/generateIsomorphicTests.ts

```diff
@@ -31,20 +32,21 @@ function isReactClass(Component: any): boolean {
  * @param Components  main export from package
  * @param props custom props per component
  * @param children custom children per component
- * @param skipList array of component names to skip
+ * @param renderSkipList array of component names to skip
  */
 export function generateIsomorphicTests(
     Components: { [name: string]: any },
     props: { [name: string]: any },
     children: { [name: string]: React.ReactNode },
-    skipList: string[] = [],
+    renderSkipList: string[] = [],
+    classNameSkipList: string[] = ["Alert", "Dialog", "MenuItem", "Popover2", "Portal", "Toaster", "Tooltip2"],
```

Comment:
remove this default value, use `[]`. set this array in `packages/core/test/isotest.js`

Reply:
Ah that makes sense! Moved all customizations to packages/core/test/isotest.js